### ISSUE TITLE
Inserting pegof's signature/fingerprint(?)

### ISSUE THREAD

**Issue by masatake** [dolik-rce/pegof#15]
For verification and debugging build-scripts utilizing pegof, I think pegof should insert the message something like

```
# Optimized code is generated by Pegof v x.y.z from input-file-name.peg
# Don't edit this file manually.
```

to at the beginning of the generated files.

**Comment by dolik-rce** [comment]
Sure, that is easy to do. But it should only happen when it runs in the optimization mode. It wouldn't make much sense when using pegof as a formatter. And even for optimization, it should be possible to turn it off via command line option.

**Comment by masatake** [comment]
Oh, I see.

**Comment by dolik-rce** [comment]
Implemented on the devel branch. There is new option `-H`/`--header` that can be used to control the behavior.

**Comment by dolik-rce** [comment]
Merged to master, the new option is available in pegof 0.5.0.

**Comment by masatake** [comment]
Thank you. I'll use the option in the ctags side.
